www-apps/gitea/gitea-1.7.1.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit golang-vcs-snapshot systemd user
+
+EGO_PN="code.gitea.io/gitea"
+KEYWORDS="~amd64 ~arm"
+
+DESCRIPTION="A painless self-hosted Git service, written in Go"
+HOMEPAGE="https://gitea.io/"
+SRC_URI="https://github.com/go-gitea/gitea/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="pam sqlite"
+
+S="${WORKDIR}/${P}/src/${EGO_PN}"
+
+PATCHES=(
+	"${FILESDIR}/fix-chpw-loop.patch"
```

Comment:
Is this accepted upstream? Why is it needed?

Reply:
Accepted upstream: https://github.com/go-gitea/gitea/commit/f4c7e87fc9c9ff0659a33affc0136b38da30e0aa in queue for the next release: 1.7.3